app/controllers/journey_controller.rb

```diff
@@ -31,12 +32,7 @@ def summary
     @event_generator = RouletteService.new(params, current_character)
     status = @event_generator.generate_travel_event
 
-    case status
-    when :dead
-      render layout: 'wide',  :locals => {:background => 'dead'}
-    when :success
-      render layout: 'wide',  :locals => {:background => 'start'}
-    end
+    render layout: 'wide',  :locals => {:background => 'start', status: status}
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.